src/components/v2/V2Project/V2PayButton/index.tsx

```diff
@@ -45,13 +46,14 @@ export default function V2PayButton({
   const isPayDisabled = Boolean(disabledMessage) || disabled
 
   return (
-    <>
+    <div style={{ textAlign: 'center', ...wrapperStyle }}>
```

Comment:
same in v2 button.

Btw we need to consolidate these buttons into a single resuable component.